process.sh

```diff
@@ -36,9 +36,7 @@ main() {
         echo 'CI: Skip push to AWS and GCS'
     else
         aws s3 cp --exclude "$OUTPUT/*.ome.tif*" --exclude "$OUTPUT/*.zarr/*" --recursive "$OUTPUT" s3://"$CLOUD_TARGET"
-        gsutil -m cp -r "$OUTPUT/spraggins/spraggins.images" "gs://$CLOUD_TARGET/spraggins/spraggins.images"
-        gsutil -m cp -r "$OUTPUT/spraggins/spraggins.ims.zarr" "gs://$CLOUD_TARGET/spraggins/spraggins.ims.zarr"
-        gsutil -m cp -r "$OUTPUT/linnarsson/linnarsson.images.zarr" "gs://$CLOUD_TARGET/linnarsson/linnarsson.images.zarr"
+        gsutil -m rsync -r -x ".*\.png$|.*\.json$" "$OUTPUT" gs://"$CLOUD_TARGET"
```

Comment:
I'd like it to be really clear that the two sets are complements. Could one be based on an exclude, and the other an include? Looking at the aws docs:

       Note that, by default, all files are included.  This means that provid-
       ing only an --include filter will not  change  what  files  are  trans-
       ferred.   --include  will only re-include files that have been excluded
       from an --exclude filter.  If you only want to upload files with a par-
       ticular extension, you need to first exclude all files, then re-include
       the files with the particular extension.  This command will upload only
       files ending with .jpg:

          aws s3 cp /tmp/foo/ s3://bucket/ --recursive --exclude "*" --include "*.jpg"

       If  you wanted to include both .jpg files as well as .txt files you can
       run:

          aws s3 cp /tmp/foo/ s3://bucket/ --recursive \
              --exclude "*" --include "*.jpg" --include "*.txt"

Unfortunately, I don't think we could use the exact same pattern, since AWS is globs, and Google is regex, but we could still make the intent obvious.

(If we really wanted to, we could probably wire up something with `find | grep | xargs`... but that wouldn't improve readability.)

Reply:
Good suggestion. Added:

```bash
aws s3 cp --recursive --exclude "*" --include "*.png" --include "*.json" $OUTPUT s3://"$CLOUD_TARGET"
gsutil -m rsync -r -x ".*\.png$|.*\.json$" $OUTPUT gs://"$CLOUD_TARGET"
```